README.md

```diff
@@ -13,38 +14,95 @@ At this time there are no web server-side components.
 
 ## DaoStack-Arc-Js
 
-Alchemy uses a library of reusable and configurable Ethereum smart contracts called "DaoStack-Arc-Js".  You can see all of the source code for those contracts and even contribute to the project [here](https://github.com/daostack/arc-js).
+Alchemy uses a library of reusable and configurable Ethereum smart contracts called "DaoStack-Arc-Js". You can see all of the source code for those contracts and even contribute to the project [here](https://github.com/daostack/arc-js).
 
 ## See It Live
-You can browse to the latest release at [http://daostack.azurewebsites.net](http://daostack.azurewebsites.net).  This deployment of Alchemy assumes you are either running a Kovan chain locally (listening on localhost:8485), or you are using Chrome and have an extension such as MetaMask that enables you to connect to a Kovan node.
 
-## Building The Code
+You can browse to the latest release at [http://daostack.azurewebsites.net](http://daostack.azurewebsites.net). This deployment of Alchemy assumes you are either running a Kovan chain locally (listening on localhost:8485), or you are using Chrome and have an extension such as MetaMask that enables you to connect to a Kovan node.
 
-To build the code, follow these steps.
+## Getting Started
 
-1. Ensure that [NodeJS](http://nodejs.org/) is installed.
-2. From the project folder, install the packages:
+First, ensure that [NodeJS](http://nodejs.org/), v8.0.0 or greater, is installed.
 
-  ```shell
-    npm install
-  ```
-3. build the application:
-  ```shell
-    npm start webpack.build.production
-  ```
+Now run the following scripts.  These will install the required packages and run several scripts that bring up ganache in a separate window,
+migrate Arc contracts to it, build the application and bring it up in your default browser:
 
-4. start the application:
-  ```shell
-    npm start serve
-  ```
+```shell
+npm install
+npm start
+```
+
+If `npm start` doesn't work for you, and for more information about the scripts involved, see the following section "Build and Run".
```

Comment:
> If `npm start` doesn't work for you 

Why ?  Is there a known cases where it will not run ?
If yes, can you list that ?
If not i guess this line should be updated :  

> For more information about the scripts involved, see the following section "Build and Run".

Reply:
I honestly am not sure which OSs will work and which won't, but it definitely won't work for all of them, thus it is anticipated that `npm start` won't work for everyone, thus I prefer to be clear about that in the comment.

If we want we can create a git issue to try to make runAsync on every OS

Reply:
But we do have a way to run async on all OS, no?
We do it in daostack in the `test` script.
It works quite well AFAIK.

Reply:
> But we do have a way to run async on all OS, no?

I could probably put something together, but would prefer to make that a separate ticket.  The instructions are there for the case when the Async script doesn't work.